src/rng.rs

```diff
@@ -0,0 +1,78 @@
+//! Pseudorandom number generators (PRNGs).
+
+/// A simple and efficient random number generator.
+pub type SimpleRng = Xorshiro256PlusPlus;
+
+/// A xorshiro256++ random number generator.
+///
+/// This is a simplified version of the `SimpleRng` implementation from the
+/// excellent `rand` crate, keeping only essential features.
+///
+/// Source: https://docs.rs/rand/0.8.4/src/rand/rngs/xoshiro256plusplus.rs.html
```

Comment:
Could we add a comment about how hard this would be to break (incl. seeding requirements)?

Reply:
What do you mean? What kind of attack? Seeding requirements are the same for any PRNG - if you know the seed, you know the outputs.

Reply:
Attack as in Codeforces hacks. (In cryptography, you might allow an even stronger attack model.) Deterministic seeds will always be vulnerable but, I'm guessing this has decent theoretical guarantees (at least in the upper bits?) if the seed has some entropy. Well, if you're not certain, it's probably best to leave it as is; you've already provided multiple references for any concerned readers to check out.

Reply:
Sweet, I just noticed your added documentation 👍

Reply:
I think you have a misunderstanding here. In Codeforces, the only "hacks" for random number generation are precisely when you can guess the seed, due to issues like not enough entropy. Random number generators can be attacked in any number of ways, but the vast majority of Codeforces attacks are simply because of guessing the seed.

Reply:
I suppose there's a difference between what empirically works and what's theoretically guaranteed. The former should be enough, but it's still nice to know which we're dealing with right? If some user decides to take this code somewhere with a threat model that we didn't intend, at least they'll know.